hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/record/AbstractReplicatedRecordStore.java

```diff
@@ -86,7 +86,7 @@ private Object remove(InternalReplicatedMapStorage<K, V> storage, Object key) {
             storage.remove(marshalledKey, current);
         }
         if (replicatedMapConfig.isStatisticsEnabled()) {
-            getStats().incrementRemoves(Clock.currentTimeMillis() - time);
+            getStats().incrementRemoves(System.nanoTime() - time);
```

Comment:
This call with nanos instead of millis as the argument automatically changes unit for `maxRemoveLatency` (and other `max*Latency` fields) in `LocalReplicatedMapStatsImpl` to nanos. Thus, they should be converted when reading the probe and when serializing stats to JSON, like it's done in `LocalMapStatsImpl`.